### PR TITLE
splits controller from main code adds some unready tests

### PR DESCRIPTION
**Pull request by mommel**

### CODE REVIEW

**Comment by mommel** [comment]
Uncomplete but might be easily adopted for Air @Illumiated

**Review by Illumiated — APPROVED** [review]
Ok will try to use the same struct for Air

File: code/Schwurbler360/src/main.cpp [hunk 1]
```diff
@@ -49,20 +47,61 @@
...
+  rotaryController->HandleMidiValueChangeCallback(&SendMidiValueChange);
+};
 
+// cppcheck-suppress unusedFunction // Function is used by arduino
 void loop() {
-  // put your main code here, to run repeatedly:
+  buttonController->GetData();
+  rotaryController->GetData();
+  delay(2000);
+  while (usbMIDI.read()) {
```
Comment (mommel) [inline code/Schwurbler360/src/main.cpp:105]: Codacy found an issue: [Check buffer boundaries if used in a loop including recursive loops (CWE-120, CWE-20).  ](https://app.codacy.com/gh/mommel/hs-lr-midi-schwurbler/pullRequest?prid=6339623)

**Review by Illumiated — APPROVED** [review]

**Comment by mommel** [comment]
OK ty

**Pull request merged**